src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/MigrateVisitService.kt

```diff
@@ -58,6 +60,10 @@ class MigrateVisitService(
   private val migrateMaxMonthsInFuture: Long,
 ) {
 
+  companion object {
+    val LOG: Logger = LoggerFactory.getLogger(this::class.java)
```

Comment:
tiny point but should this be called log in lowercase?

Reply:
Well I see this as constant and constant should be in upper case.